src/test/java/org/minions/devfund/damianvp/WhackAMoleTest.java

```diff
@@ -0,0 +1,196 @@
+package org.minions.devfund.damianvp;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class to test the WhackAMole Class.
+ */
+public class WhackAMoleTest {
+
+    public static final int ATTEMPTS = 50;
+    public static final int GRID_DIMENSION = 10;
+    private WhackAMole whackAMoleGame;
+
+    /**
+     * Instance WhackAMole class in order to start a new Game.
+     */
+    @Before
+    public void startNewGame() {
+        whackAMoleGame = new WhackAMole(ATTEMPTS, GRID_DIMENSION);
+    }
+
+    /**
+     * Test place Mole using valid locations.
+     */
+    @Test
+    public void testPlaceMoleValidPositions() {
+
+        final int xLocation = 0;
+        final int yLocation = 0;
+        int expectedMolesLeft = 1;
+        assertTrue(whackAMoleGame.place(xLocation, yLocation));
+        assertEquals(expectedMolesLeft, whackAMoleGame.getMolesLeft());
+    }
+
+    /**
+     * Test place Mole using negative locations.
+     */
+    @Test
+    public void testPlaceMoleUsingNegativeLocations() {
+//        WhackAMole game = startNewGame(4,4);
+        final int xLocation = -1;
+        final int yLocation = -1;
+        int expectedMolesLeft = 0;
+        assertFalse(whackAMoleGame.place(xLocation, yLocation));
+        assertEquals(expectedMolesLeft, whackAMoleGame.getMolesLeft());
+//        assertTrue(whackAMoleGame.place());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)